File: src/main/marmara/model/WeeklyDate.py
class WeeklyDate:
    def __init__(self, day_name, hours):
        self.day_name = day_name
        self.hours = hours

    def __str__(self):
        return f"WeeklyDate{{ day_name={self.day_name}, hours={self.hours} }}"

    def get_day_name(self):
        return self.day_name

    def set_day_name(self, day_name):
        self.day_name = day_name

    def get_hours(self):
        return self.hours

    def set_hours(self, hours):
        self.hours = hours

File: java/src/main/java/edu/marmara/model/WeeklyDate.java
package edu.marmara.model;

public class WeeklyDate {
    private DayName dayName;
    private Integer hours;

    public WeeklyDate() {
    }

    public WeeklyDate(DayName dayName, Integer hours) {
        this.dayName = dayName;
        this.hours = hours;
    }

    @Override
    public String toString() {
        return "WeeklyDate{" +
                "dayName=" + dayName +
                ", hours=" + hours +
                '}';
    }

    public DayName getDayName() {
        return dayName;
    }

    public void setDayName(DayName dayName) {
        this.dayName = dayName;
    }

    public Integer getHours() {
        return hours;
    }

    public void setHours(Integer hours) {
        this.hours = hours;
    }
}
